### DISCUSSION TITLE
Can we run Lighthouse CI in Snapshot mode?

### DISCUSSION THREAD

**Discussion by andre-fong** [GoogleChrome/lighthouse-ci#932] [Q&A] [answered]
I've been trying to run Lighthouse to test for accessibility against a Single Page Application where user action is needed before running any Lighthouse checks. Does Lighthouse CI have an option to run in snapshot or timespan mode?

### Context

We could go ahead and test this using Snapshot mode in the [Lighthouse Node API](https://github.com/GoogleChrome/lighthouse/blob/main/docs/user-flows.md), but I haven't been able to find a way to do this using Lighthouse CI. I need to use Lighthouse CI for the [LHCI server support](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/server.md) which can store reports on a remote server over time. 

I'm aware that [Lighthouse CI can run a puppeteer script](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#collect) before running, but it is only used for auth as stated in the docs (the page will refresh only saving any relevant cookies and not the needed user input).

**Comment by adamraine** [comment, marked as answer]
Currently no, user flow support is tracked here https://github.com/GoogleChrome/lighthouse-ci/issues/383

**Reply by andre-fong** [reply]
Sorry must've missed that, thanks!
